script_editor/app/views/plays/show.html.erb

```diff
@@ -321,36 +321,38 @@
 
         down = false
         toCut = false
+        parent = null
         // all words will need to be printed within the div class "script-main"
 
-        function mouseDown(e) {
-            if (e.srcElement.className == "word" || e.srcElement.className == "punc" ||
-                e.srcElement.className == "stage") {
-                    startIndex = lowIndex = highIndex = Array.from(e.srcElement.parentNode.children)
-                                                             .indexOf(e.srcElement)
+        function mouseDown({ srcElement }) {
```

Comment:
Does this have to be e.srcElement, or does JavaScript know srcElement refers to the source srcElement of the event listener?